runtime/wasm/bigstring.wat

```diff
@@ -201,7 +201,29 @@
                (local.set $len (i32.sub (local.get $len) (i32.const 1)))
                (local.set $pos (i32.add (local.get $pos) (i32.const 1)))
                (br $loop))))
-     (ref.i31 (i32.const -1)))
+      (ref.i31 (i32.const -1)))
+
+   (func (export "caml_bigstring_memrchr")
```

Comment:
Does this belong here ? Or in another repo ?

Reply:
Since I may change the implementation of big arrays at some point, I think it's simpler to put this here for now.